### DISCUSSION TITLE
Cant generate API key with enterprise

### DISCUSSION THREAD

**Discussion by michaellifering** [codemagic-ci-cd/codemagic-docs#2102] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Have enterprise account
Go to App store connect
Look for API Keys tab

### Expected results

To see API keys tab

### Actual results

There is no tab.
I am admin.

**Comment by himesh-cm** [comment]
Hi @michaellifering,

Apple does not provide an App Store API for the Apple Enterprise Developer program. As a result, publishing and code signing must be done manually.

For code signing - You can find our sample workflow [here](https://github.com/codemagic-ci-cd/codemagic-sample-projects/blob/main/ios/ios-manual-code-signing-demo-project/codemagic.yaml).

**Comment by himesh-cm** [comment]
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
